QubitServer/src/main/java/org/labrad/qubits/channeldata/AnalogData.scala

```diff
@@ -0,0 +1,9 @@
+package org.labrad.qubits.channeldata
+
+import org.labrad.qubits.channels.AnalogChannel
+
+trait AnalogData extends Deconvolvable {
+  def setChannel(channel: AnalogChannel): Unit
+  def getDeconvolved(): Array[Int]
```

Comment:
is it more standard to leave off the empty-parens? I guess the more standard thing tod do would be to name it `deconvolvedData` or something.